fedmsg_migration_tools/bridges.py

```diff
@@ -14,20 +14,18 @@
 # You should have received a copy of the GNU General Public License along
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
-from __future__ import absolute_import
 
+import json
 import logging
-import logging.config
 
-from kombu import Connection, Exchange, Queue
-from kombu.mixins import ConsumerMixin
-from kombu.pools import connections
+import fedora_messaging.api
```

Comment:
I tend to use the ``from fedora_messaging import api, message`` form, but I suppose we should at least make ``api`` and ``message`` modules available in ``fedora_messaging.__init__.py``. Or maybe just ``api`` and expose the ``Message`` class in there?

Reply:
Definitely agree about style, I just feel bad making style comments that aren't caught automatically by linters. I try to live by whatever flake8 lives by unless it's an egregious break in style :smile: